### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by blerude** [jplisk03/prj-rev-bwfs-tea-cozy#5]
Grade: Exceeds Expectations

Great work on this project! You've successfully included all of the necessary elements and your code is generally well formatted and easy to understand. This is helped by the comments you left to clarify the role of each code block. You have a clear understanding of how to apply classes and ids to HTML elements in order to style them, and your CSS is solid. 

Moving forward, I encourage you to use commenting on your CSS doc as well. You should also be extra careful to avoid duplicating pieces of code unnecessary. Those are the details that you need to focus on as a programmer. Also, feel free to give your CSS some of your own style. I recommend using Google Fonts to find interesting fonts, and Coolors.co to generate color palettes that will enhance the visual aspect even more. Again, great job here!

**Comment by jplisk03** [comment]
Thank you for the feedback. Ill have to look back in my notes on how to use commenting on CSS. 
For the personalizing features would I have to down load those new fonts and colors?

Lastly, I was having some trouble figuring out how to get the black boxes around the locations and also getting the header to had the logo in the upper left and links inside the header line like the sample website shows. 
I also had the similar trouble trying to get the columns right in the previous project which was to recreate the newspaper website.

**Comment by blerude** [comment]
Nope, Coolors.co just gives you hex codes to use in the CSS (color: #xxxxxx vs. color: 'white'), and Google Fonts gives you a link to include in the header of your HTML file that enables the app to use whichever font you select. 

For the borders around locations, add a 'border' and 'borderRadius' to the .location styling. You can adjust the size/color/etc. as you please. You can use `<a>` tags with href="link that you want to redirect to" to turn the header into a link. I hope that helps!
